scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -52,11 +53,13 @@ workflow GvsQuickstartIntegration {
     String effective_gatk_docker = select_first([gatk_docker, GetToolVersions.gatk_docker])
     String effective_hail_version = select_first([hail_version, GetToolVersions.hail_version])
 
-    call FilterIntervalListChromosomes {
-        input:
-            full_interval_list = full_wgs_interval_list,
-            chromosomes = ["chrX", "chrY", "chr20"],
-            variants_docker = effective_variants_docker,
+    if (chr20_X_Y_only) {
+        call FilterIntervalListChromosomes {
+            input:
+                full_interval_list = full_wgs_interval_list,
+                chromosomes = ["chrX", "chrY", "chr20"],
+                variants_docker = effective_variants_docker,
+        }
```

Comment:
what was the reasoning behind adding this as an option?

Reply:
I ran into a failure that I thought was due to Nirvana not being happy about all the missing chromosomes. Unfortunately I can't seem to find the error I ran into in job history so I'm going to re-make the Quickstart VDS with only 20, X and Y to make sure that was really the problem.

Reply:
OK here was the problem: we split up our processing by chromosomes, and Nirvana was [mightily displeased](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/69258dbd-2341-4e11-a128-b85c766af792) to find no genes associated with the missing chromosomes (see most `AnnotateVCF` shards):

```
ERROR: The provided TSV has no valid custom annotation entries.
```

Reply:
ah, ok — thanks for finding that out!